numpy/core/_exceptions.py

```diff
@@ -5,6 +5,7 @@
 By putting the formatting in `__str__`, we also avoid paying the cost for
 users who silence the exceptions.
 """
+from numpy.core.overrides import set_module
```

Comment:
Perhaps we should find a better place for set_module?

Reply:
Probably. At least `overrides.py` does not create any circular imports yet.